src/Codec/Xlsx/Lens.hs

```diff
@@ -84,7 +84,7 @@ ixCellRC i = wsCells . ix i
 -- | lens to access cell in a worksheet using more traditional
 -- x+y coordinates
 ixCellXY :: (Int, Int) -> Traversal' Worksheet Cell
-ixCellXY = ixCellRC . swap
+ixCellXY i = ixCellRC $ swap i
```

Comment:
any hint why GHC 9 fails to compile it in point-free form?

Reply:
This is by design: https://gitlab.haskell.org/ghc/ghc/-/wikis/migration/9.0#simplified-subsumption
TLDR: to implement impredicativity they had to sacrifice some inference. In this case polymorhpic types do not unify anymore.

The error is:

```
src/Codec/Xlsx/Lens.hs:87:12: error:
    • Couldn't match type ‘c2’ with ‘Traversal' Worksheet Cell’
      Expected: (Int, Int) -> c2
        Actual: (Int, Int) -> Traversal' Worksheet Cell
      Cannot instantiate unification variable ‘c2’
      with a type involving polytypes: Traversal' Worksheet Cell
```